browser/js/common/directives/navbar/navbar.html

```diff
@@ -1,30 +1,78 @@
-<nav class="navbar navbar-static-top">
-    <div class="container">
-        <ul>
-            <li>
-                <a ng-click="goToSection('.problem')"> The Problem </a>
-            </li>
+<nav id='navbar' class='navbar navbar-static-top'>
+	<div class='container'>
+		<div class='dropdown'>
+			<div class='dropdown-title'>
+				What is Kinetic
+			</div>
+				<div class='dropdown-content'>
+					<div class='list-container'>
+						<a ng-click="goToSection('.problem', 'home')"> The Problem </a>
+						<a ng-click="goToSection('.solution', 'home')"> Our Solution </a>
+						<a ng-click="goToSection('.what-is', 'home')"> What We Do </a>
+						<a ng-click="goToSection('.about-us', 'home')"> About Us </a>
+						<a ng-click="goToSection('.sponsored', 'home')"> Key Issues </a>
+						<a ng-click="goToSection('.get-involved', 'home')"> Get Involved </a>
+					</div>
+				</div>
+		</div>
```

Comment:
this is less to do with the actual code, but more with the overall implementation. i think it would make a lot more sense to put a `data` attribute on each of the `a` tags equal to the class name that you want to scroll to, and put a `data` attribute on the containing `div` equal to the state you want to go to. then attach a click handler to each `a` tag and onclick, look at the data on the parent `div` to get the state to go to and look at the data on the `a` element to figure out which section, and then do the stuff. i think that makes it a lot easier to extend and makes the code a lot cleaner.